python_modules/dagster/dagster/_core/definitions/definitions_class.py

```diff
@@ -740,3 +742,111 @@ def with_reconstruction_metadata(self, state_metadata: Mapping[str, Any]) -> Sel
                 **state_metadata,
             },
         )
+
+    @public
+    def map_asset_specs(self, fn: Callable[[AssetSpec], AssetSpec]) -> "Definitions":
```

Comment:
IMO for now we should make this a standalone function and non-public API and use it ourselves so we have the option value to change things. Even just a week of real world dogfooding would make me more comfortable.

Reply:
👍 made that change